site/docs/pages/getting-started.mdx

```diff
@@ -30,6 +30,48 @@ bun add @coinbase/onchainkit
 
 :::
 
+### API Key
```

Comment:
Let's say `Public API Key`, as it is a public one

Reply:
Updated all instanced to include the public prefix